examples/scale.cpp

```diff
@@ -45,21 +45,22 @@ int main()
     }
 
     // If you need to reset the scaling in s (our Scale object), then you can do this:
-    s.autoscale_from (vf2); // will immediately autoscale from the container of values vf2.
+    s.compute_scaling_from (vf2); // will immediately compute the scaling function from the container of values vf2.
```

Comment:
```suggestion
    s.compute_scaling_from_data (vf2); // will immediately compute the scaling function from the container of values vf2.
```
Suggestion of name. could also be `compute_scaling_from_container`, just to make it clear that it is not `from_min_max`

Reply:
Yes, I think this is a good clarification. I'll do that.